src/scss/_base.scss

```diff
@@ -1,80 +1,58 @@
-////
-/// @group o-table
-/// @link http://registry.origami.ft.com/components/o-table
-////
-
-/// Base tables styles - add to <table>
+/// Base tables styles and simple utility classes.
 @mixin oTableBase() {
-	@include oTypographyBody;
-	border-collapse: collapse;
-	border-spacing: 0;
-	margin: 0 0 20px;
-	width: 100%;
-
-	caption {
+	.o-table {
+		@include oTypographyBody;
+		border-collapse: collapse;
+		border-spacing: 0;
 		margin: 0;
-		padding: 0;
 		width: 100%;
-	}
-
-	th {
-		@include oTypographySansBold(1);
-		color: _oTableGet('table-header-color');
-		background-color: _oTableGet('header-background');
-	}
 
-	&[data-o-table--js] thead th:not([data-o-table-heading-disable-sort]) {
-		cursor: pointer;
-		user-select: none;
-		padding-right: 20px;
-		&:after {
-			@include oIconsGetIcon('arrows-up-down', $container-width: 20,  $container-height: 20, $iconset-version: 1);
-			content: '';
-			margin-right: -20px;
+		th,
+		td {
+			padding: 10px;
+			text-align: left;
 			vertical-align: top;
 		}
-		// Show descending icon with DSC sort applied.
-		&[aria-sort='descending'] {
-			&:after {
-				@include oIconsGetIcon('arrow-down', $container-width: 20,  $container-height: 20, $iconset-version: 1);
-				vertical-align: top;
+
+		th {
+			@include oTypographySansBold(1);
```

Comment:
[`oTypographyBold('sans')`](https://github.com/Financial-Times/o-typography/blob/079112ec83b024da2a22e714ee5dba0be871a427/src/scss/_mixins.scss#L149) doesn't rewrite the font-size or font-family

Reply:
(But this only works if oTypographySans has already been applied to the element, and we're applying individually to `ths` and `tds` here?)